docassemble/CLAGuardianship/data/questions/petition_for_appointment_of_guardian.yml

```diff
@@ -162,7 +162,7 @@ code: |
 comment: |
   This block includes the logic for standalone interviews.
   Delete mandatory: True to include in another interview
-mandatory: True
+# mandatory: True
```

Comment:
I don't think this is the right fix. Maybe instead, remove line 8 of the petition for removal? It shouldn't be including another interview that has its own launching point.

Reply:
@nonprofittechy I just pushed my latest changes that allow us to keep `mandatory: True` in both the appointment and removal files.